src/components/structures/RightPanel.js

```diff
@@ -31,7 +31,6 @@ module.exports = React.createClass({
     displayName: 'RightPanel',
 
     propTypes: {
-        userId: React.PropTypes.string, // if showing an orphaned MemberInfo page, this is set
```

Comment:
I don't think so given that the file panel and member list take the room as a prop. I suppose the RightPanel could listen to `view_room` instead of accepting roomID it as a prop?